example/views/index.haml

```diff
@@ -229,9 +229,13 @@
                       %dt.heading{ data: { value: "Published On" } }
                       %dt.field{ data: { value: "published_on" } }
                       %dt.sortable{ data: { value: "published_on" } }
+                      %dt.tooltipContent{ data: { value: "<div style='text-align: left;'>This is tooltip<br /><b>High:</b> Test<div>".html_safe }}
```

Comment:
ideally the heading - data - value is HTML
and add one props:
```
%dt.heading-type{ data: { value: 'html'}}
```

Reply:
i make it the headingType to match the camel case we use on react